datatree/datatree.py

```diff
@@ -244,6 +244,7 @@ class DataTree(
     TreeAttrAccessMixin,
     Generic[Tree],
     Mapping,
+    TreeNode,
```

Comment:
This addition doesn't change anything because we are already inheriting from `NamedNode`, which inherits from `TreeNode`.

Reply:
yah. i have observed that, in NamedNode. Thanks for the clarification.